common/data_refinery_common/models/models.py

```diff
@@ -279,6 +280,13 @@ def save(self, *args, **kwargs):
         if not self.id:
             self.created_at = current_time
         self.last_modified = current_time
+
+        if self.accession_code and not self.alternate_accession_code:
+            if self.accession_code.startswith('GSM'):
+                self.alternate_accession_code = 'E-GEOD-' + self.accession_code[3:]
+            elif self.accession_code.startswith('E-GEOD-'):
+                self.alternate_accession_code = 'GSM' + self.accession_code[7:]
```

Comment:
This file uses `GSM` and the migration uses `GSE`. What's going on with that? Should that be tested?